src/common/snippets/include/snippets/pass/insert_loops.hpp

```diff
@@ -19,12 +19,13 @@ namespace pass {
 class InsertLoops: public ngraph::pass::FunctionPass {
 public:
     OPENVINO_RTTI("InsertLoops", "0");
-    InsertLoops(ov::PartialShape master_shape, size_t vector_size);
+    InsertLoops(ov::PartialShape master_shape, size_t loop_depth, size_t vector_size);
```

Comment:
1. Elaborated description: added all parameters, not only loop_depth. BTW loop_depth == tileRank, I renamed since we don't have tiles anymore
2. `vector_size` is like an `increment`, but only for the innermost loop, and only if we have enough data. So it's more like a `maximum possible increment` than just `increment`. I can change to `max_increment`, but I would rather keep `vector_size`, since it also explains the reason for our limitation.